### DISCUSSION TITLE
useFetch how to get error message?

### DISCUSSION THREAD

**Discussion by mihail727** [nuxt/nuxt#17825] [Questions] [answered]
I have read now `const {error} = await useFetch(api)` returns only boolean.
I also saw some solution to get error text:
```ts
if (error.value) {
	if (process.client) {
		await refresh();
	}
	console.log(error.value.toString());
	return [];
}
```
But I would also like to get the object that the server sends at the time of the error:
`@/server/api/get-items.ts`
```ts
export default async (req: IncomingMessage, res: ServerResponse) => {
	try {
		// some code
	} catch (error) {
		res.statusCode = 400;
		res.end(JSON.stringify({ message: error.message })); // this i want to get on the client side
	}
};
```

**Comment by mihail727** [comment, marked as answer]
@danielroe sorry for tag.... can you help please xD

**Reply by danielroe** [reply]
Follow https://github.com/nuxt/framework/issues/2122 for updates.

**Reply by mihail727** [reply]
so there is no solution at the moment?
